.github/workflows/pages.yml

```diff
@@ -0,0 +1,23 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - documentation_dev
+    tags: '*'
+  pull_request:
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: '1.7'
+      - name: Install dependencies
+        run: julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.add("Documenter"); Pkg.instantiate()'
```

Comment:
I don't understand why but I have to specify `Pkg.add("Documenter");` here, or the CI won't recognize `Documenter` package. But glad it's working.

Reply:
it's because we don't necessarily want Documenter to be a part of the codebase for MRIGradients.jl. So in order to make things work, we add it manually in the CI